actions/actionsRoutes.js

```diff
@@ -0,0 +1,88 @@
+const express = require('express');
+const dbActions = require('../data/helpers/actionModel');
+const router = express.Router();
+
+router.get('/', (req, res) => {
+  dbActions
+    .get()
+    .then((actions) => {
+      res.status(200).json(actions);
+    })
+    .catch((err) => {
+      res.status(500).json({ error: "The information could not be retrieved." })
+    });
+});
+
+router.get("/:id"), (req, res) => {
+  dbActions
+    .get(req.params.id)
```

Comment:
You can add a check here to see if the id matches with an existing action in the database.